.github/workflows/badge_or_last_milestone.yml

```diff
@@ -81,3 +87,26 @@ jobs:
             As usual, please let us know if you run into any delays by leaving a comment on the application PR, or directly submitting an
             [amendment](https://github.com/w3f/Grants-Program#changes-to-a-grant-after-approval).
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    
+  comment-last-milestone:
+    runs-on: ubuntu-latest
+    needs: check-needs-badge
+    if: needs.check-needs-badge.outputs.was_last_milestone
+    steps:
+      - name: Last milestone comment
+        uses: thollander/actions-comment-pull-request@1.0.2
+        with:
+          message: > 
+            We noticed that this is the last milestone of your project, so what are the potential next steps?<br/>
+            
+            While you can always apply for a follow-up grant, the grants program might not be the best tool for your future growth. 
+            Our main goal is to support research as well as early-stage technical projects. Below are a couple of other options in our ecosystem 
+            that might be better suited as the next step: <br/> 
+
+            Project with a Business Case/Token: [Substrate Builders Program](https://substrate.io/ecosystem/substrate-builders-program/) 
+            or [VC Funding](https://parachains.info/investors) <br/> 
+
+            Common Good Projects: [Treasury Funding](https://github.com/w3f/Grants-Program#treasury) <br/> 
+
+            Let us know if you have any questions regarding the above. We are more than happy to help you with it. 
```

Comment:
```suggestion
            Let us know if you have any questions regarding the above. We are more than happy to help you determine the best course of action. 
```

Reply:
Maybe this is going too far but I figure we can at least help them determine some of the differences between funding sources.

Reply:
No. I think this makes sense for teams that completed a grant. Thanks!